src/modules/organizations/model.js

```diff
@@ -31,6 +25,8 @@ export const organizationAvatarShape = React.PropTypes.shape({
   src: React.PropTypes.string
 });
 
+export const organizationsAvatarsShape = React.PropTypes.arrayOf(organizationAvatarShape);
```

Comment:
I think it's because there are multiple organizations each with it's own avatar, so also multiple avatars, therefore it was named `organizationsAvatars`, as opposed to `organizationAvatars` which might be interpreted to represent many avatars from a single organization, which would be misleading? Not sure that makes sense, but I think that was the original rationale.